### PR TITLE
[FEAT] #19 - Implement common method to detect request country

### LINKED ISSUE #15: [FEAT] Data modeling for multiple langauge
**Description**
We have to save multiple language(english/korean). So, remodeling for support multiple langugage.

**Additional context**
I think below model is suitable

``` text plain
IndieApp     IndieAppDetail        Country
=========   ==================   =========
IndieAppId   IndieAppDetailId      CountryId
- etc -      IndieAppId            CountryName
             CountryId             Language
             Name                   - etc -
             DetailDescription
              - etc -
```
* Reference: [https://stackoverflow.com/questions/929410/what-are-best-practices-for-multi-language-database-design](https://stackoverflow.com/questions/929410/what-are-best-practices-for-multi-language-database-design)

### PR DESCRIPTION
**Pull request by sgh002400**
- Implement GetLocale.java to get accept-language
- Verify GetLocale.getLocale() through test

### CODE REVIEW

File: src/test/java/opgg/weba/JamPick/common/GetLocaleTest.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class GetLocaleTest {
+
+    @Test
+    public void GetAcceptLanguage() {
+
+        Assertions.assertThat(GetLocale.getLocale().toString()).isEqualTo("ko_KR");
```
Comment (sgh002400) [inline src/test/java/opgg/weba/JamPick/common/GetLocaleTest.java:14]: You can use also
`Assertions.assertThat(GetLocale.getLocale()).isEqualTo(Locale.KOREA);`

File: src/main/java/opgg/weba/JamPick/common/GetLocale.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import org.springframework.stereotype.Component;
+
+import java.util.Locale;
+
+@Component
+public class GetLocale {
+
+    public static Locale getLocale() {
+
+        Locale locale = LocaleContextHolder.getLocale();
```
Comment (gmlwo530) [inline src/main/java/opgg/weba/JamPick/common/GetLocale.java]:
```suggestion

```

File: src/main/java/opgg/weba/JamPick/common/GetLocale.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        return locale;
```
Comment (gmlwo530) [inline src/main/java/opgg/weba/JamPick/common/GetLocale.java]:
```suggestion
        return LocaleContextHolder.getLocale();
```

how about this?

File: src/test/java/opgg/weba/JamPick/common/GetLocaleTest.java (hunk 1 above)
Comment (gmlwo530) [reply src/test/java/opgg/weba/JamPick/common/GetLocaleTest.java:14]: As i know, `GetLocale` return accept-language of request header. But test hasn't test code about http request or request header. Is it okay?

File: src/main/java/opgg/weba/JamPick/common/GetLocale.java (hunk 3 above)
Comment (sgh002400) [reply src/main/java/opgg/weba/JamPick/common/GetLocale.java]: Thank you for advice :)

File: src/test/java/opgg/weba/JamPick/common/GetLocaleTest.java (hunk 1 above)
Comment (sgh002400) [reply src/test/java/opgg/weba/JamPick/common/GetLocaleTest.java:14]: I agree with that. I'll modify logic. thanks!

Comment (sgh002400) [reply src/test/java/opgg/weba/JamPick/common/GetLocaleTest.java:14]: There were some problems when testing with Mockmvc.
So i made TestController that return only locale value and test with postman.

Result is below

image

But, i won't add TestController.

Comment (gmlwo530) [reply src/test/java/opgg/weba/JamPick/common/GetLocaleTest.java:14]: It is okay that edit this test in future. If you want to do other task at first, merge this PR.

I approved PR, thanks

**Review by gmlwo530 — APPROVED** [review]

**Pull request merged**